.devops/code-review-pipelines.yml

```
@@ -12,7 +12,7 @@ resources:
     - repository: pagopaEcommerceLocal
       type: github
       name: pagopa/pagopa-ecommerce-local 
-      ref: main
+      ref: CHK-3316-align-env
```

```suggestion
      ref: main
```

TODO: to be restored after dependent ecommerce-local PR merge